src/Network/Network.Test/ScenarioTests/NetworkTestRunner.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using Microsoft.Azure.Commands.TestFx;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Commands.Network.Test.ScenarioTests
+{
+    public class NetworkTestRunner
+    {
+        protected readonly ITestRunner TestRunner;
+
+        protected NetworkTestRunner(ITestOutputHelper output)
+        {
+            TestRunner = TestFx.TestManager.CreateInstance (output)
+                .WithNewPsScriptFilename ($"{GetType().Name}.ps1")
+                .WithProjectSubfolderForTests ("ScenarioTests")
+                .WithCommonPsScripts (new[]
+                {
+                    @"Common.ps1",
+                    @"../AzureRM.Resources.ps1",
+                    @"../AzureRM.Storage.ps1",
+                })
+                .WithNewRmModules (helper => new[]
+                {
+                    helper.RMProfileModule,
+                    helper.GetRMModulePath("AzureRM.Monitor.psd1"),
+                    helper.GetRMModulePath("AzureRM.Network.psd1"),
+                    helper.GetRMModulePath("AzureRM.Compute.psd1"),
+                    helper.GetRMModulePath("AzureRM.ContainerInstance.psd1"),
+                    helper.GetRMModulePath("AzureRM.OperationalInsights.psd1"),
```

Comment:
Recently we had to [add Storage reference](https://github.com/Azure/azure-powershell/blob/master/src/Network/Network.Test/NetworkResourcesController.cs#L132) to make some tests pass in Live mode. Do we need the same there?

Reply:
To use Storage basic functionality we add the [script file](https://github.com/Azure/azure-powershell/blob/481c209b06ff0bbdf05c898226d758844b4c16f9/src/Network/Network.Test/ScenarioTests/NetworkTestRunner.cs#L20).

Reply:
@vladimir-shcherbakov, I've tried to run tests on this branch (Record mode) and it looks like the test using storage fails. However in case I add AzureRM.Storage.psd1 to this list, it still fails with the same error:
```
  <Results>
    <UnitTestResult executionId="6ed3ba66-daaa-4eda-aa0c-b6ca361696e2" testId="b28595f8-3a5e-64f2-d7eb-8f54b5c7b3f4" testName="Commands.Network.Test.ScenarioTests.NetworkWatcherAPITests.TestTroubleshoot" computerName="EA" duration="00:31:32.2290000" startTime="2019-01-30T14:24:17.4768810+03:00" endTime="2019-01-30T14:24:17.4768815+03:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="6ed3ba66-daaa-4eda-aa0c-b6ca361696e2">
      <Output>
        <StdOut>Test-Troubleshoot&#xD;
Caught Exception: System.Management.Automation.CommandNotFoundException: The term 'New-AzStorageContext' is not recognized as the name of a cmdlet, function, script file, or operable program.&#xD;
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.&#xD;
```

Should I update something to make this test pass?

Reply:
@EvgenyAgafonchikov 
Did you try the live tests on the master branch?

Reply:
We will either need to enhance the storage.ps1 or add the storage library to this solution and include Storage.psd1 here.  Either way, since there is a separate item to enable runnign the unit tests from the branch, we can defer this untill after the controller is replaced.

Reply:
I discussed this offline with @EvgenyAgafonchikov : 
> Completed checking: test passes after update like this (fail w/o it)
helper.GetRMModulePath("AzureRM.Storage.psd1"),